pkg/chain/ethereum/config.go

```diff
@@ -45,6 +45,14 @@ type Config struct {
 	// allowed gas price is reached, no further resubmission attempts are
 	// performed.
 	MaxGasPrice uint64
+
+	// RequestsPerSecondLimit sets the maximum average number of requests
+	// per second which can be executed against the Ethereum node.
```

Comment:
It wasn't clear to me when reviewing the code we want to rate-limit all types of requests. It makes a perfect sense but let's make it clear in the documentation:
```
// All types of chain requests are rate-limited, including view function calls.
```